.travis.yml

```diff
@@ -21,3 +21,4 @@ script:
  - test/check-style
  - ./wait-for-api || { docker ps; docker logs cacophony-api; exit 1; }
  - cd test && pytest
+ - docker exec -it cacophony-api /app-isolated/node_modules/.bin/sequelize db:migrate:undo --config /app-isolated/config/app_test_default.js --migrations-path /app-isolated/migrations
```

Comment:
This appears to undo the latest DB migration. Is that particularly useful without a more detail check afterwards?